### ISSUE TITLE
Text sizes - move customisations.css to style.css

### ISSUE THREAD

**Issue by spleggenkuhle** [campai-bv/autotask-dashboards-os#74]
Is there any way to change the text/number/value sizes for any of the Labels, Legends or any text sizes at all?

**Comment by coencoppens** [comment]
Of course! Just put on some gloves and get ready go get your hands dirty :-) Everything is changeable through the css file /app/webroot/css/customisations.css.

I'm adding this as a feature to the roadmap as well. It would be nice to have all default styles in style.css and use the customisations.css to allow you to change it. That would prevent you from losing your changes with future updates.

**Comment by spleggenkuhle** [comment]
Any guidance as to where within that .css file to find the text size settings for the ques or the various graphs that are available?

**Comment by coencoppens** [comment]
The widget titles are all H4's, but I suggest you use a developer toolbar to easily identify the objects you want to modify.
